README.markdown

```diff
@@ -217,52 +218,29 @@ The example above demonstrates the following style guidelines:
 
 ### Use of Self
 
-For conciseness, avoid using `self` since Swift does not require it to access an object's properties or invoke its methods.
-
-Use `self` when required to differentiate between property names and arguments in initializers, and when referencing properties in closure expressions (as required by the compiler):
+Always use `self` when possible. Although Xcode's syntax highlighting makes distinguishing easier, the use of `self` makes it even easier. The distinguising comes in with local variables vs properties, and global functions vs instance/class functions/methods:
```

Comment:
IMO we should take advantage of not having to `self`` properties & methods. :-1:  (<-- geez, i feel bad about putting a thumbs down on anything)